src/main/java/uk/gov/laa/pfla/auth/service/services/MappingTableService.java

```diff
@@ -23,7 +31,8 @@ public MappingTableService(MappingTableDao mappingTableDao) {
         this.mappingTableDao = mappingTableDao;
     }
 
-    public List<ReportListResponse> createReportListResponseList() {
+    public List<ReportListResponse> createReportListResponseList() throws Exception {
+        mappingTableDao.setupDB();
```

Comment:
Can we avoid calling setupDB each time?

Reply:
That would make sense, yeh i'll look to see how else to arrange the flow